pkg/apis/flow/v1alpha1/flow.proto

```diff
@@ -103,40 +103,43 @@ message Kubernetes {
 
   string source_pod_namespace = 2;
   string source_pod_name = 3;
-  bytes source_pod_uid = 4;
+  string source_pod_uid = 4;
   Labels source_pod_labels = 5;
 
   string source_node_name = 6;
-
-  string destination_pod_namespace = 7;
-  string destination_pod_name = 8;
-  bytes destination_pod_uid = 9;
-  Labels destination_pod_labels = 10;
-
-  string destination_node_name = 11;
-
-  bytes destination_cluster_ip = 12;
-  uint32 destination_service_port = 13;
-  string destination_service_port_name = 14;
-  bytes destination_service_uid = 15;
-
-  NetworkPolicyType ingress_network_policy_type = 16;
-  string ingress_network_policy_namespace = 17;
-  string ingress_network_policy_name = 18;
-  bytes ingress_network_policy_uid = 19;
-  string ingress_network_policy_rule_name = 20;
-  NetworkPolicyRuleAction ingress_network_policy_rule_action = 21;
-
-  NetworkPolicyType egress_network_policy_type = 22;
-  string egress_network_policy_namespace = 23;
-  string egress_network_policy_name = 24;
-  bytes egress_network_policy_uid = 25;
-  string egress_network_policy_rule_name = 26;
-  NetworkPolicyRuleAction egress_network_policy_rule_action = 27;
-
-  string egress_name = 28;
-  bytes egress_ip = 29;
-  string egress_node_name = 30;
+  string source_node_uid = 7;
```

Comment:
@tnqn @luolanzone I have changes all "UID" fields from `string` to `bytes`. This was done for a couple of reasons:
- this is the type used by the K8s API
- I think it's better to convert to a byte slice in the FlowAggregator, just before IPFIX export
- if needed we add `_uuid` alternative fields (with type `bytes`) in the future to "replace" these ones
- while this representation uses 2x the space, in the future I would like to reduce this impact by only sending names / UIDs / etc. in the first flow record (as they don't change during the lifetime of the connection)

Reply:
yes, that's right. Let me edit the comment.